common.mk

```diff
@@ -372,8 +374,21 @@ LIBBLIS_SO_PATH    := $(BASE_LIB_PATH)/$(LIBBLIS_SO)
 # library lists its soname as 'libblis.so.n', where n is the .so major version
 # number, a symlink in BASE_LIB_PATH is needed so that ld can find the local
 # shared library when the testsuite is run via 'make test' or 'make check'.
-LIBBLIS_SO_MAJ_PATH := $(BASE_LIB_PATH)/$(LIBBLIS_SO).$(SO_MAJOR)
 
+ifeq ($(OS_NAME),Darwin)
+LIBBLIS_SONAME := $(LIBBLIS).$(SO_MAJOR).$(SHLIB_EXT)
+else ifeq ($(findstring MSYS,$(OS_NAME)),MSYS)
```

Comment:
@isuruf The naming `libblis.1.dylib` seems off for Darwin. Is that really the format, and not `libblis.dylib.1`? (I don't use OS X, so I have to defer to others here.)

Reply:
`libblis.1.dylib` is the norm on OSX.

Reply:
Okay, thanks, good to know.